package.json

```diff
@@ -20,12 +20,6 @@
     "prettier": "^2.5.1",
     "turbo": "=1.10.3"
   },
-  "lint-staged": {
-    "*.{ts,tsx,js,jsx}": [
-      "eslint",
-      "prettier --write"
-    ]
-  },
```

Comment:
removed because it is duplicated.
the `eslint` does not have --fix option and `prettier --write` command needs to be before the `eslint` command